code/src/Components/CoordsImageDisplay.js

```diff
@@ -0,0 +1,43 @@
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+import React from 'react';
+import hej from './coord1-0.png';
+import hej10 from './coord1-1.png';
+import hej11 from './coord1-2.png';
+import hej12 from './coord1-3.png';
+import hej13 from './coord1-4.png';
+import hej14 from './coord1-5.png';
+import hej15 from './coord1-6.png';
+
+export const CoordsImageDisplay = () => {
+  const coordinates = useSelector((store) => store.labyrinthMango.coordinates)
+  let imageToShow = null;
+
+  if (coordinates === '0,0') {
```

Comment:
This was a nice way of displaying different images depending on where in the labyrinth you were. I see that you've set the imageToShow to null first. We returned the whole function (with switch case) as the background image. I'm wondering if this way (that you've done), of making the image a variable, instead might be a better way performance-wise.